src/prefect/settings.py

```diff
@@ -482,6 +482,23 @@ def default_cloud_ui_url(settings, value):
 # Setting definitions
 
 
+PREFECT_LIMIT_EXCEPTION_TO_STATE_MESSAGE = Setting(
+    bool,
+    default=False,
+)
+"""If `True`, exception messages are suppressed from State Messages and not sent over
+the internet. An exception type is still included. Users will need to rely on logging
```

Comment:
Could you remove language from the doc strings (or doc string once consolidated) about "sent over the internet"